site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -75,44 +82,40 @@ spec:
     kind: ClusterIssuer
 EOF
 
-# Optional: Store the secret ca.crt in a file to be used by the client for the server authentication
-TLS_CA_FILE=ca.crt
-TLS_CA_VALUE=`kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}'`
-echo ${TLS_CA_VALUE} | base64 -d > ${TLS_CA_FILE}
-
-# In case of MacOS
-# echo ${TLS_CA_VALUE} | base64 -D > ${TLS_CA_FILE}
-
 # Add ca.crt to the allocator-tls-ca Secret
+TLS_CA_VALUE=`kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}'`
 kubectl get secret allocator-tls-ca -o json -n agones-system | jq '.data["tls-ca.crt"]="'${TLS_CA_VALUE}'"' | kubectl apply -f -
 ```
 
 ## Client Certificate
 
-Because agones-allocator uses an mTLS authentication mechanism, client must provide a certificate that is accepted by the server. Here is an example of generating a client certificate. For the agones-allocator service to accept the newly generate client certificate, the generated client certificate CA or public portion of the certificate must be added to a kubernetes secret called `allocator-client-ca`.
+Because agones-allocator uses an mTLS authentication mechanism, client must provide a certificate that is accepted by the server. 
+
+If Agones is installed using Helm, you can leverage a default client secret, `allocator-client.default`, created in the game server namespace and allowedlisted in `allocator-client-ca` k8s secret. You can extract and use that secret for client side authentication, by following the example in [the allocation sample]({{< relref "#send-allocation-request" >}}).
```

Comment:
"allowedlisted" - one word, two? I'm actually not sure.

Maybe "allowlisted" is better? Felt like one too many "ed"?